build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -234,15 +241,8 @@ private void GenerateRuntimeGraph(DotNetCli dotnetCli, string destinationDeps)
             }
         }
 
-        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot, string hostFxrVersion)
+        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot)
         {
-            File.Copy(
-                Path.Combine(_corehostLockedDirectory, HostArtifactNames.DotnetHostBaseName),
```

Comment:
> Can the version be specified through the command line args?

Invoking dotnet.exe that is sibling to the _shared_ folder (which contains M.N.App\<version>) with _-fx-version_ argument and specifying the version will enable the same behavior.